lib/WebAPI/DBIC/Resource/ActiveModel/Role/DBIC.pm

```diff
@@ -150,7 +155,12 @@ sub render_activemodel_response { # return top-level document hashref
 sub render_item_as_activemodel_hash {
     my ($self, $item) = @_;
 
-    my $data = $self->render_item_as_plain_hash($item);
+    my $data = {
+        # I would rather use $self->activemodel_type_for_class($item->result_class),
+        # but type_namer isn't defined when we get here via:
+        # SetWritable->create_resources_from_active_model
+        $self->result_key => $self->render_item_as_plain_hash($item),
```

Comment:
@timbunce any thoughts on why type_namer is undefined at this point in the call stack?  I can send you a stack trace if that would be helpful.  Thanks.